vault/resource_aws_secret_backend.go

```diff
@@ -159,8 +159,8 @@ func awsSecretBackendResource() *schema.Resource {
 		consts.FieldPath,
 		consts.FieldType,
 		consts.FieldDescription,
-		consts.FieldDefaultLeaseTTL,
-		consts.FieldMaxLeaseTTL,
+		consts.FieldDefaultLeaseTTLSeconds,
+		consts.FieldMaxLeaseTTLSeconds,
```

Comment:
This looks like a breaking change. If we want we can deprecate the old and add the new. Otherwise, we probably want to revert this change.